src/redux/books/bookslice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+// STATE
+const initialBooksState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
```

Comment:
- [ ] This was the initial code for your `bookSlice`, you were meant to continue building out your `bookslice` from this file and not create a new one entirely. But since you have created a new file, I'll suggest you delete this as it is of no use to the project anymore. :+1: :100: 

- [ ] ALso the reducer.js file in the redux folder appears to be empty and of no use. I'll suggest you delete it. :small_airplane: :wave: